cli/install/template.go

```diff
@@ -473,9 +474,6 @@ data:
       # drop all labels that we just made copies of in the previous labelmap
       - action: labeldrop
         regex: __meta_kubernetes_pod_label_linkerd_io_proxy_(.+)
-      # __meta_kubernetes_pod_label_foo=bar => foo=bar
```

Comment:
Yeah, I totally agree that this could be useful down the road. If we do decide to bring it back, we might want to investigate some sort of safeguard to ensure that arbitrary labels don't blow up our label cardinality in prometheus.